decidim-courses/app/forms/decidim/courses/admin/course_form.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Courses
+    module Admin
+      # A form object used to create courses from the admin
+      # dashboard.
+      #
+      class CourseForm < Form
+        include TranslatableAttributes
+
+        mimic :course
+
+        translatable_attribute :description, String
+        translatable_attribute :instructors, String
+        translatable_attribute :title, String
+
+        attribute :hashtag, String
+        attribute :promoted, Boolean
+        attribute :scope_id, Integer
+        attribute :scopes_enabled, Boolean
+        attribute :show_statistics, Boolean
+        attribute :slug, String
+
+        attribute :address, String
+        attribute :course_date, Decidim::Attributes::LocalizedDate
+        attribute :duration, Integer
+        attribute :modality, String
+        attribute :registration_link, String
+
+        attribute :banner_image
+        attribute :hero_image
+        attribute :remove_banner_image
+        attribute :remove_hero_image
+
+        validates :scope, presence: true, if: proc { |object| object.scope_id.present? }
+        validates :slug, presence: true, format: { with: Decidim::Course.slug_format }
+
+        validate :slug_uniqueness
+
+        validates :modality, inclusion: { in: ::Decidim::Course::MODALITIES }
```

Comment:
```suggestion
        validates :modality, presence: true, inclusion: { in: ::Decidim::Course::MODALITIES }
```

Reply:
I'm developing with docker, and the image used in the project ( `decidim/decidim:0.22.0`) have the version `2.6` inside it. I'll revert the change to the `Gemfile.lock` and try to not push this change again.